.gitignore

```diff
@@ -1,4 +1,5 @@
 build/
+build_cori/
```

Comment:
I'm leaving a comment about the line that changed. Looks good!

Reply:
Hi @cemitch99, let's also add `build_perlmutter/`.

A general way to write this could be: `build_*/`